diffgraph/structured_export.py

```
@@ -0,0 +1,795 @@
+        kind = 'commit_range'
+        base_ref = parts[0]
+        head_ref = parts[1]
+    else:
+        kind = 'unstaged'
+        base_ref = None
+        head_ref = None
+
+    # repo_root
+    try:
+        result = subprocess.run(
+            ['git', 'rev-parse', '--show-toplevel'],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)